.github/workflows/test.yml

```
@@ -0,0 +1,46 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Test tfx-bsl
+
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - master
```

```suggestion
  pull_request:
```

I think we probably don't want to trigger on pushes to any branch, otherwise pushes to branches that don't have PRs yet will be launching CI jobs.

---

Agreed, I had a TODO to remove this, but missed it.